File: integration-tests/hibernate-reactive-panache/src/main/java/io/quarkus/it/panache/reactive/Bug7721OtherEntity.java
package io.quarkus.it.panache.reactive;

import jakarta.persistence.Entity;

import io.quarkus.hibernate.reactive.panache.PanacheEntity;

@Entity
public class Bug7721OtherEntity extends PanacheEntity {
    public String foo;

    public void setFoo(String foo) {
        this.foo = foo.toUpperCase();
    }
}

File: integration-tests/hibernate-reactive-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/reactive/kotlin/Bug7721OtherEntity.kt
package io.quarkus.it.panache.reactive.kotlin

import io.quarkus.hibernate.reactive.panache.kotlin.PanacheEntity
import jakarta.persistence.Entity
import java.util.Locale

@Entity
class Bug7721OtherEntity : PanacheEntity() {
    var foo: String? = null
        set(value) {
            field = value?.uppercase(Locale.getDefault())
        }
}
